SwiftGen.playground/Pages/Plist-Demo.xcplaygroundpage/Sources/Implementation Details.swift

```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+// MARK: - Implementation Details
+
+public func arrayFromPlist<T>(at path: String) -> [T] {
+  guard let url = bundle.url(forResource: path, withExtension: nil),
+    let data = NSArray(contentsOf: url) as? [T] else {
+    fatalError("Unable to load PLIST at path: \(path)")
+  }
+  return data
+}
+
+public struct PlistDocument {
+  let data: [String: Any]
+
+  public init(path: String) {
+    guard let url = bundle.url(forResource: path, withExtension: nil),
+      let data = NSDictionary(contentsOf: url) as? [String: Any] else {
+        fatalError("Unable to load PLIST at path: \(path)")
+    }
+    self.data = data
+  }
+
+  public subscript<T>(key: String) -> T {
+    guard let result = data[key] as? T else {
+      fatalError("Property '\(key)' is not of type \(T.self)")
+    }
+    return result
+  }
+}
+
+public var bundle: Bundle!
```

Comment:
Funny that you put this one at the end while the same bundle var is at the top in the other (the JSON-Demo one)

Reply:
Ah, must have missed that one, I was moving those around at the end to have them in the same place.